src/webhook/mutation/pod_mutator.go

```diff
@@ -794,3 +800,24 @@ func silentErrorResponse(podName string, err error) admission.Response {
 	rsp.Result.Message = fmt.Sprintf("Failed to inject into pod: %s because %s", podName, err.Error())
 	return rsp
 }
+
+// getSecretConfigData fetches the SecretConfig (initSecret.Config) from the pod namespace
+func (m *podMutator) getSecretConfigData(nsName string) (*standalone.SecretConfig, error) {
+	var scs corev1.Secret
+	err := m.apiReader.Get(context.TODO(), client.ObjectKey{Name: dtwebhook.SecretConfigName, Namespace: nsName}, &scs)
+	if err != nil {
+		podLog.Error(err, "Failed to fetch config secret", "name", dtwebhook.SecretConfigName, "namespace", nsName)
+		return nil, err
+	}
+
+	scj := []byte(scs.Data[standalone.SecretConfigFieldName])
+
+	var sc standalone.SecretConfig
```

Comment:
```suggestion
	var config standalone.SecretConfig
```

Reply:
Missed this in the first commit, as the msg was hidden.

Fixed in latest commit.